### PR TITLE
Gradient offset

### PR DESCRIPTION
**Pull request by benoneal**

### CODE REVIEW

File: packages/rug/src/values.js [hunk 1]
```diff
@@ -209,6 +208,10 @@ export const opacity = {
  * Shadows
  */
 
+const shadow = (color) => (
```
Comment (lukebrooker) [inline packages/rug/src/values.js]: Can we leave this in utils?

There doesn't seem to be a big difference in shadow vs shadows at this stage.

File: lerna.json [hunk 2]
```diff
@@ -10,5 +10,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.0.0"
+  "version": "2.0.0"
```
Comment (lukebrooker) [inline lerna.json]: I have more breaking changes…

So we don't care if this keeps going up super quick?

**Review by lukebrooker — DISMISSED** [review]
Shadow changes would be good. Or just leave shadow as it was.

File: lerna.json (hunk 2 above)
Comment (benoneal) [reply lerna.json]: why would we care? is there some virtue to having low numbers?

Comment (lukebrooker) [reply lerna.json]: That's why I was asking, just to see if we care. All good if not.

**Pull request merged**